core/src/main/java/com/novoda/noplayer/drm/provision/ProvisionExecutor.java

```diff
@@ -25,7 +23,7 @@ public static ProvisionExecutor newInstance() {
         this.capabilities = capabilities;
     }
 
-    public byte[] execute(MediaDrm.ProvisionRequest request) throws IOException, UnableToProvisionException {
+    public byte[] execute(ModularDrmProvisionRequest request) throws IOException, UnableToProvisionException {
         if (isIncapabaleOfProvisioning()) {
```

Comment:
I know you guys didn't introduce this, but it would be good to fix the typo here 😇